internal/manifests/mutate.go

```diff
@@ -59,24 +66,31 @@ func MutateFuncFor(existing, desired client.Object) controllerutil.MutateFn {
 	}
 }
 
+func mergeWithOverride(dst, src interface{}) {
+	err := mergo.Merge(dst, src, mergo.WithOverride)
+	if err != nil {
+		panic(err)
```

Comment:
We should certainly not panic here. We should handle errors properly.

Reply:
fixed